packages/core/src/textures/TextureGCSystem.js

```diff
@@ -61,6 +61,11 @@ export class TextureGCSystem extends System
      */
     postrender()
     {
+        if (!this.renderer.renderingToScreen)
+        {
+            return;
+        }
+
```

Comment:
do you think this could cause issue where textures will never be removed if I for example am rendering my scene directly to a render texture each frame then using that as an input?

Reply:
Input for what? You have to do readPixels() for that. readPixels()-bound app has much bigger problems than that ;)

Reply:
ah i see! So it would get called when I render the input to the screen :D